src/db/models/bascket.py

```diff
@@ -0,0 +1,18 @@
+from datetime import datetime
+from sqlalchemy import Column, DateTime, Integer, String, JSON, ForeignKey, Numeric
+from sqlalchemy.orm import Relationship
+from db.conection import Base
+
+class Bascet(Base):
+    __tablename__ = "bascet"
+    
+    id = Column(Integer, primary_key=True)
+    user_id = Column(Integer, ForeignKey('users.id'), nullable=False)
+    smartphone_id = Column(Integer, ForeignKey('smartphones.id'), nullable=False)
+    smartphones = Column(JSON, nullable=False)
```

Comment:
Why we need 2 similar colums for smartphones?

Reply:
associations created